### PR TITLE
Registration page

### PR DESCRIPTION
**Pull request by sorayamoss**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <META HTTP-EQUIV="Content-type" CONTENT="text/html; charset=UTF-8">
```
Comment (dborstelmann) [inline public/index.html]: porque?

File: src/components/Person.vue [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
-      <p>{{ company }}</p>
-      <p>{{ bio }}</p>
-    </div>
+        <el-card shadow="hover" :body-style="{ padding: '0px' }">
+          <div class="image-wrapper">
+            <img :src="imgSource" class="image">
+          </div>
+          <div style="padding: 10px;">
+            <span class="name">{{ name }}</span>
+            <div class="bottom clearfix">
```
Comment (dborstelmann) [inline src/components/Person.vue]: clearfix for what?

File: src/components/Registration.vue [hunk 3]
```diff
@@ -0,0 +1,111 @@
+<template>
+  <div class="register">
+  <el-card class="box-card">
+    <div slot="header" class="clearfix">
+      <span>Register</span>
+    </div>
+    <el-form ref="form" :model="form" action="https://webto.salesforce.com/servlet/servlet.WebToLead?encoding=UTF-8" method="POST" label-width="120px">
```
Comment (dborstelmann) [inline src/components/Registration.vue]: salesforce?

File: src/components/Registration.vue [hunk 4]
```diff
@@ -0,0 +1,111 @@
+<template>
+  <div class="register">
+  <el-card class="box-card">
+    <div slot="header" class="clearfix">
```
Comment (dborstelmann) [inline src/components/Registration.vue]: why clearfix?

File: src/components/Registration.vue [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+        <el-input v-model="form.title"></el-input>
+      </el-form-item>
+      <el-form-item label="Email">
+        <el-input v-model="form.email"></el-input>
+      </el-form-item>
+      <el-form-item label="Twitter">
+        <el-input v-model="form.twitter"></el-input>
+      </el-form-item>
+      <el-form-item >
+        <el-checkbox label="Remember Me" v-model="form.checked" name="type"></el-checkbox>
```
Comment (dborstelmann) [inline src/components/Registration.vue]: How are we doing this?  I wouldn't introduce unnecessary complexity

File: src/views/Registration.vue [hunk 6]
```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="registration">
+    <el-container>
+      <el-header class="header">
+        <img class="logo" src="../assets/slalom-logo-blue-RGB.jpg"/>
+      </el-header>
+      <el-main class="main">
+        <el-row>
+          <el-col :span="12"><div class="grid-content bg-purple"><h1 class="title">Welcome to HealthSTLX</h1></div></el-col>
+          <el-col :span="12"><div class="grid-content bg-purple-light"><Registration class="registration-form"/></div></el-col>
```
Comment (dborstelmann) [inline src/views/Registration.vue]: maybe put registration on it's own line

**Comment by dborstelmann** [comment]
Also can you post some screenshots of what the page looks like?

File: src/components/Registration.vue (hunk 5 above)
Comment (sorayamoss) [reply src/components/Registration.vue]: Also from Derek's example salesforce form, can take out if we do not want/ can't figure out how to do it

File: src/views/Registration.vue (hunk 6 above)
Comment (sorayamoss) [reply src/views/Registration.vue]: can do

**Comment by sorayamoss** [comment]
screen shot 2018-07-09 at 11 30 26 am
screen shot 2018-07-09 at 11 30 59 am

File: src/components/Person.vue (hunk 2 above)
Comment (dborstelmann) [reply src/components/Person.vue]: Go read an article about clearfix and then I’ll leave it up to you about whether to keep it.  It has a very specific meaning/use case

File: src/components/Registration.vue (hunk 4 above)
Comment (dborstelmann) [reply src/components/Registration.vue]: See above

File: src/components/Registration.vue (hunk 5 above)
Comment (dborstelmann) [reply src/components/Registration.vue]: Yeah I would take it out for now

**Comment by dborstelmann** [comment]
Good enough for me!  Just get those flix fixed

**Pull request merged**